numpy/distutils/system_info.py

```diff
@@ -2409,6 +2409,7 @@ def get_config_exe(self):
         return self.default_config_exe
 
     def get_config_output(self, config_exe, option):
+        import subprocess
```

Comment:
There's an `import subprocess as sp` above in this module, conditional on platform. Maybe there's some historical stuff I don't know about `subprocess` availability but I suppose we could just import at top level these days?

Reply:
We `import subprocess` all over distutils, lets move this import to the top.